templates/aside_settings.html.hbs

```diff
@@ -0,0 +1,30 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
This should probably move into a `<head>`

Reply:
On second thought, it looks like most of the html templates have font-awesome in their `<head>` already, so this should probably just be removed

Reply:
Yeah, that should have been removed after I tested it (I have a separate local folder for doing visual checks x.X).